packages/react/accordion/src/Accordion.stories.tsx

```diff
@@ -677,7 +676,3 @@ const contentAttrClass = css({
   display: 'block',
   ...styles,
 });
-
-export function Accordion2(props: AccordionSingleProps) {
-  return <Accordion {...props} type="single" collapsible />;
-}
```

Comment:
@benoitgrelard Looks like you may have added this accidentally when you were debugging the issues Pedro was having?

Reply:
ah yeah my bad!